### PR TITLE
OP-655 Use interface instead of concreate class

### PR DESCRIPTION
**Pull request by dbmalkovsky**
See OP-655

As usual some formatting is included

### CODE REVIEW

File: src/main/java/org/isf/lab/gui/LabNew.java [hunk 1]
```diff
@@ -417,7 +418,7 @@ private JPanel getJPanelResults() {
                 resultsContainerScroll.setVerticalScrollBarPolicy(ScrollPaneConstants.VERTICAL_SCROLLBAR_AS_NEEDED);
                 resultsContainerScroll.setBounds(0, 0, EAST_WIDTH, RESULT_HEIGHT);
                 jPanelResults.add(resultsContainerScroll);
-                ArrayList<ExamRow> exaRowArray;
+				List<ExamRow> exaRowArray;
```
Comment (mwithi) [inline src/main/java/org/isf/lab/gui/LabNew.java:421]: indentation problem

File: src/main/java/org/isf/opd/gui/OpdEdit.java [hunk 2]
```diff
@@ -674,7 +675,7 @@ public void keyTyped(KeyEvent e) {}
             searchDiseaseButton.setPreferredSize(new Dimension(20, 20));
             searchDiseaseButton.setIcon(new ImageIcon("rsc/icons/zoom_r_button.png"));
             searchDiseaseButton.addActionListener(new ActionListener() {
-                ArrayList<Disease> diseasesOPD = null;
+	            List<Disease> diseasesOPD = null;
```
Comment (mwithi) [inline src/main/java/org/isf/opd/gui/OpdEdit.java:678]: indentation problem

File: src/main/java/org/isf/opd/gui/OpdEdit.java [hunk 3]
```diff
@@ -794,7 +795,7 @@ public void keyTyped(KeyEvent e) {}
             searchDiseaseButton3.setPreferredSize(new Dimension(20, 20));
             searchDiseaseButton3.setIcon(new ImageIcon("rsc/icons/zoom_r_button.png"));
             searchDiseaseButton3.addActionListener(new ActionListener() {
-                ArrayList<Disease> diseasesOPD = null;
+	            List<Disease> diseasesOPD = null;
```
Comment (mwithi) [inline src/main/java/org/isf/opd/gui/OpdEdit.java:798]: intendation problem?

File: src/main/java/org/isf/priceslist/gui/PricesBrowser.java [hunk 4]
```diff
@@ -84,25 +85,25 @@
...
-	private ArrayList<PriceList> listArray;
-	private ArrayList<Price> priceArray;
+	private List<PriceList> listArray;
+	private List<Price> priceArray;
 	private PriceList listSelected;
 	
 	private PriceNode examNodes;
 	private ExamBrowsingManager examManager = Context.getApplicationContext().getBean(ExamBrowsingManager.class);
-    private ArrayList<Exam> examArray;
+    private List<Exam> examArray;
```
Comment (mwithi) [inline src/main/java/org/isf/priceslist/gui/PricesBrowser.java:94]: indentation problem?

**Review by mwithi — CHANGES_REQUESTED** [review]
Small changes required

File: src/main/java/org/isf/lab/gui/LabNew.java (hunk 1 above)
Comment (dbmalkovsky) [reply src/main/java/org/isf/lab/gui/LabNew.java:419]: Fixed

File: src/main/java/org/isf/opd/gui/OpdEdit.java (hunk 2 above)
Comment (dbmalkovsky) [reply src/main/java/org/isf/opd/gui/OpdEdit.java]: Fixed

File: src/main/java/org/isf/opd/gui/OpdEdit.java (hunk 3 above)
Comment (dbmalkovsky) [reply src/main/java/org/isf/opd/gui/OpdEdit.java]: Fixed

File: src/main/java/org/isf/priceslist/gui/PricesBrowser.java (hunk 4 above)
Comment (dbmalkovsky) [reply src/main/java/org/isf/priceslist/gui/PricesBrowser.java]: Fixed

**Review by mwithi — APPROVED** [review]

**Pull request merged**